src/main/resources/data-h2.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO alcohol (id,name,cost,amount) VALUES (1,'Tatra', 2, 10);
```

Comment:
You are inserting and defining id value are you sure this id will not be generated by  h2 ?